app-admin/rex/rex-1.13.1.ebuild

```diff
@@ -0,0 +1,203 @@
+	dev-perl/Dist-Zilla-Plugin-CheckExtraTests
+	dev-perl/Dist-Zilla-Plugin-ContributorsFile
+	dev-perl/Dist-Zilla-Plugin-Git
+	dev-perl/Dist-Zilla-Plugin-Git-Contributors
+	dev-perl/Dist-Zilla-Plugin-MakeMaker-Awesome
+	dev-perl/Dist-Zilla-Plugin-Meta-Contributors
+	dev-perl/Dist-Zilla-Plugin-MetaProvides-Package
+	dev-perl/Dist-Zilla-Plugin-NextVersion-Semantic
+	dev-perl/Dist-Zilla-Plugin-OSPrereqs
+	dev-perl/Dist-Zilla-Plugin-OurPkgVersion
+	dev-perl/Dist-Zilla-Plugin-Run
+	dev-perl/Software-License
+"
+RDEPEND="
+	!minimal? (
+		dev-perl/DBI
+		dev-perl/Expect
+		dev-perl/IPC-Shareable
+		dev-perl/XML-LibXML
+	)
+	virtual/perl-Carp
+	virtual/perl-Data-Dumper
+	dev-perl/Data-Validate-IP
+	dev-perl/Devel-Caller
+	dev-perl/Digest-HMAC
+	virtual/perl-Digest-MD5
+	virtual/perl-Exporter
+	virtual/perl-File-Spec
+	dev-perl/HTTP-Message
+	dev-perl/Hash-Merge
+	virtual/perl-IO
+	dev-perl/IO-String
+	dev-perl/IO-Tty
+	dev-perl/JSON-MaybeXS
+	virtual/perl-MIME-Base64
+	dev-perl/Net-OpenSSH
+	dev-perl/Net-SFTP-Foreign
+	virtual/perl-Scalar-List-Utils
+	dev-perl/Parallel-ForkManager
+	dev-perl/Sort-Naturally
+	dev-perl/String-Escape
+	virtual/perl-Storable
+	dev-perl/TermReadKey
+	virtual/perl-Test-Simple
+	dev-perl/Text-Glob
+	virtual/perl-Text-Tabs+Wrap
+	virtual/perl-Time-HiRes
+	dev-perl/URI
+	dev-perl/XML-Simple
+	dev-perl/libwww-perl
+	dev-perl/YAML
+	virtual/perl-version
+"
+
+BDEPEND="
+	${RDEPEND}
+	>=virtual/perl-CPAN-Meta-Requirements-2.120.620
+	>=virtual/perl-ExtUtils-MakeMaker-7.110.100
+	>=dev-perl/File-ShareDir-Install-0.60.0
+	virtual/perl-Module-Metadata
+	test? (
+		virtual/perl-File-Temp
+		dev-perl/Test-Deep
+		dev-perl/Test-Output
+		dev-perl/Test-UseAllModules
+		virtual/perl-autodie
+	)
+"
+
+[[ ${PV} == 9999 ]] && BDEPEND+=" ${DZIL_DEPENDS}"
+
+src_unpack() {
+	if [[ $PV == 9999 ]]; then
+		"${VCS_ECLASS}"_src_unpack
+		mkdir -p "${S}" || die "Can't make ${S}"
+	else
+		default
+	fi
+}
+
+dzil_src_prep() {
+	einfo "Patching dist.ini"
+
+	# This block of sed invocations removes all plugins that aren't
+	# useful for users to have on Gentoo, because all of them are
+	# conditional and subjective style checks, which don't indicate
+	# a real issue for users, and paying the price of their dependencies is undesired.
+
+	# The {N;d} trick adds the [n]ext line after the match to the pattern-space
+	# so that the final [d]elete deletes the next line too. Can be expanded for each
+	# line, ie: {N;N;N;d} deletes 3 lines after the match as well as the match.
+	sed -e '/^\[Test::Kwalitee\]/d' \
+		-e '/^\[PodSyntaxTests\]/d' \
+		-e '/^Perl::Critic::Freenode =/d' \
+		-e '/^Perl::Critic::TooMuchCode =/d' \
+		-e '/^Test::Kwalitee =/d' \
+		-e '/^Test::PerlTidy =/d' \
+		-e '/^Test::Pod =/d' \
+		-e '/^\[Test::CPAN::Changes\]/{N;d}' \
+		-e '/^\[OptionalFeature/,/^$/d' \
+		-e '/^\[Test::MinimumVersion\]/{N;d}' \
+		-i dist.ini || die "Can't patch dist.ini"
+}
+dzil_env_setup() {
+	# NextRelease noise :(
+	mkdir -p ~/.dzil/
```

Comment:
I would add a `|| die` here but it's a nitpick. I will add it before merging the PR.

Reply:
Right, that wouldn't hurt :+1: I'll make sure to merge back any change to my overlay too.

Reply:
> I will add it before merging the PR.

I hope you added it in all the files ... because there was "no real diff" involving that line of code. Only stupid github showing copied files as "vanilla diffs".

Reply:
If I see correctly, it was only added to the 1.13.1 ebuild for now.

I'll add it to the next iteration of the live ebuild, so when we cut the next releases based on that, the future ones include it too.

Is there any substantial value to add it retrospectively to older ebuilds as well? Want me to do a new PR for that?

Reply:
I'm still gonna go and have to to properly finish my audit and my fixes, which like, now I have even more pressure on.

No new PR is required, just gets annoying when people can't even review PR's properly, and make changes based on reviewing the PR wrong. 

I feel like taking @gentoo/proxy-maint out of the loop just to discourage people who don't understand what they're doing from doing things.

Reply:
No objections from me, you're the best to handle ebuilds like this. Just don't leave @ferki hanging with his contributions ;)

(I specifically ignored this PR for a while hoping @kentfredric would deal with it)
((to my taste the ebuild still has many QA issues, but nitpicking...))

Reply:
> Is there any substantial value to add it retrospectively to older ebuilds as well? Want me to do a new PR for that?

No, zero. The code in question that was patched *CAN ONLY* run when $PV == 9999. 

So the patching did nothing, and it didn't fix the potential problem in the one place it *could* run.

Fixed now in -9999 as well. https://github.com/gentoo/gentoo/commit/0112abb1935e4582dea0ed1d433528a748826db1

But backporting it is a big waste of time. lol